tests/Neo.UnitTests/Ledger/UT_MemoryPool.cs

```diff
@@ -736,74 +736,6 @@ public void TestUpdatePoolForBlockPersisted()
             _unit.VerifiedCount.Should().Be(0);
         }
 
-
-        [TestMethod]
-        public async Task Malicious_OnChain_Conflict()
```

Comment:
@AnnaShaleva This PR allows malicious onChain conflics?

Reply:
No, of course not. The case is that this test was constructed incorrectly from the very beginning (if you remember, it's your test from https://github.com/neo-project/neo/pull/2818#issuecomment-1632278255, and it was failing due to some balance problems, see the https://github.com/neo-project/neo/pull/2818#issuecomment-1632330064). I've fixed this test way back ago in https://github.com/neo-project/neo/pull/2818 and I've moved it to the `UT_Blockchain.cs` (see the https://github.com/neo-project/neo/commit/0c06c915ef381cf146ea5314b6a9920f8fd07b26) commit. But I've accidently forgot to remove the unworking stub from the mempool testset. So I did this in the current PR.

Check it out, the relevant `TestMaliciousOnChainConflict()` is still in the `UT_Blockchain.cs` and it passes OK.